File: core/server/project/shine/src/main/java/com/home/shine/constlist/UDPCommandType.java
package com.home.shine.constlist;

/** udp执行类型 */
public class UDPCommandType
{
	/** 发包 */
	public static final int Pack=1;
	/** 请求包 */
	public static final int RequestPack=2;
	/** 出错(包等待超上限)(被动关闭) */
	public static final int Error=3;
	/** 主动关闭 */
	public static final int Close=4;
	/** 连接 */
	public static final int Connect=5;
	/** 连接成功 */
	public static final int ConnectSuccess=6;
}

File: core/client/game/src/shine/constlist/UDPCommandType.cs
namespace ShineEngine
{
	public class UDPCommandType
	{
		/** 发包 */
		public const int Pack=1;
		/** 请求包 */
		public const int RequestPack=2;
		/** 出错(包等待超上限)(被动关闭) */
		public const int Error=3;
		/** 主动关闭 */
		public const int Close=4;
		/** 连接 */
		public const int Connect=5;
		/** 连接成功 */
		public const int ConnectSuccess=6;
	}
}